src/main/java/com/minecolonies/coremod/entity/ai/item/handling/ItemStorage.java

```diff
@@ -1,6 +1,7 @@
 package com.minecolonies.coremod.entity.ai.item.handling;
 
 import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'net.minecraft.item.ItemStack'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)